### DISCUSSION TITLE
Can AGH get around blockadblock?

### DISCUSSION THREAD

**Discussion by angrynerd** [AdguardTeam/AdGuardHome#3374] [Q&A]
Hello, is there a way that AGH can get around blockadblock or similar programs? I tried unblocking all of the domains that the website running blockadblock generated but somehow it was still being detected. From what I know this is solved already in uBlock and Brave but Im interested in an AGH fix for mobile devices as they cant make use of browser extensions.

**Comment by ameshkov** [comment]
Not really, getting around this kind of scripts requires more flexibility that DNS-level blocking provides.
